.github/workflows/python-client-tests.yml

```diff
@@ -0,0 +1,45 @@
+name: Python Client Tests
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+permissions:
+  contents: read
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
+    - name: Set up JDK 17
+      uses: actions/setup-java@v4
+      with:
+        java-version: '17'
+        distribution: 'temurin'
+
+    - name: Set up Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: '3.9'
+
+    - name: Generate Python client library
+      run: build/sbt pythonClient/generate
+
+    - name: Install dependencies
+      run: |
```

Comment:
I agree that some steps here are one time setups and should not be a in a script. thats what the `pip installs` are. but there are like 3 steps that should be done every time a test has to be run. 
- build/sbt pythonClient/generate
- pip install clients/python/target/
- pytest clients/python/tests
these can be combined into a single python script. when a developer is changing the api, then having to run 3 commands every time is annoying. iteratively running update api + run one command is better... isnt it?

Reply:
Sounds good, I added a simple script to run those three commands